### DISCUSSION TITLE
Not getting events from my Maschine MK3

### DISCUSSION THREAD

**Discussion by berkeleymalagon** [djipco/webmidi#260] [Q&A]
I'm using the quick start example and I'm not able to get MIDI events from my Maschine MK3. I see the device in the device list so I know webmidi can see it, and the MK3 is in MIDI mode.

I'm new to the library and mostly new to MIDI - can anyone give me a tip on debugging this?

**Comment by djipco** [comment]
Can you share the actual code you are using and the console output? The quick start example picks up the first found device by default which may or may not be your Maschine Mk3 device.

**Reply by berkeleymalagon** [reply]
In this case, it turns out it was an issue with how the MK3 exposes its virtual inputs. My other midi controller (Komplete Kontrol M32) works fine. Thank you!
